frontend/demo/component/button/button-tertiary-inline.ts

```diff
@@ -0,0 +1,22 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+
+import { html, LitElement, customElement } from 'lit-element';
+import '@vaadin/vaadin-button/vaadin-button';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('button-tertiary-inline')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  render() {
+    return html`
+      <!-- tag::snippet[] -->
+      <vaadin-button theme="tertiary-inline">Tertiary Inline</vaadin-button>
```

Comment:
```suggestion
      <vaadin-button theme="tertiary-inline">Tertiary inline</vaadin-button>
```

"Sentence case" is used for example UI texts.

Reply:
Done.